### PR TITLE
Bump react router

### PR DESCRIPTION
**Pull request by tu55eladd**

### CODE REVIEW

File: src/app.tsx [hunk 1]
```diff
@@ -54,6 +62,7 @@ const Router = ({ fnr, children }: { fnr: string; children: React.ReactNode }) =
     }
 
     const basename = getBasename(fnr);
+    console.log({ basename, fnr });
```
Comment (marcjas) [inline src/app.tsx]: >:(

File: src/moduler/aktivitet/rediger/EndreAktivitet.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    | { status: string; avtalt: boolean }
+    | IJobbAktivitetFormValues;
+
+interface SubComponentProps<Aktivitet extends VeilarbAktivitet> {
+    onSubmit: (values: FormValues) => Promise<void>;
+    dirtyRef: MutableRefObject<boolean>;
+    aktivitet: Aktivitet;
+}
+
+type EndreForm<Aktivitet extends VeilarbAktivitet> = FunctionComponent<SubComponentProps<Aktivitet>>;
```
Comment (marcjas) [inline src/moduler/aktivitet/rediger/EndreAktivitet.tsx:65]: ubrukt?

File: src/moduler/aktivitet/rediger/EndreAktivitet.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+interface FormProps<T extends AlleAktiviteter> {
```
Comment (marcjas) [inline src/moduler/aktivitet/rediger/EndreAktivitet.tsx:67]: T extends VeilarbAktivitet ?

**Review by marcjas — APPROVED** [review]

**Comment by tu55eladd** [comment]
bump-react-router

**Pull request closed without merging**